notebooks/geo/DEM_MapVisualization.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p>That might help to tell that a popup window will open and that one need to click 'image' button to see the map.</p><p> </p><p>Is there a way to zoom out ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>